pyproject.toml

```diff
@@ -15,3 +15,13 @@ npm = ["jlpm"]
 
 [tool.check-manifest]
 ignore = ["share/jupyter/labextensions/@jupyterlite/xeus-python-kernel/**", "yarn.lock", ".*", "package-lock.json", "src/xpython_wasm.js", "src/xpython_wasm.wasm", "src/*.data", "src/*.js", "src/worker.ts", "src/web_worker_kernel.ts", "*.sh"]
+
+[tool.jupyter-releaser.options]
+skip = ["check-links"]
+
+[tool.jupyter-releaser.hooks]
+before-build-npm = [
+    "python -m pip install jupyterlab~=3.1 empack>=2.0.9,<3 jupyter_packaging~=0.10,<2",
```

Comment:
This is probably because the `v1` used to install the Python package in editable mode before with `pip install -e .`